src/naive/date.rs

```diff
@@ -1054,6 +1054,49 @@ impl NaiveDate {
         )
     }
 
+    /// Subtracts another `NaiveDate` from the current date.
+    /// Returns a `MonthsDelta` of the number of calendar months between the two dates.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use chrono::{NaiveDate, MonthsDelta, Months};
+    ///
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2022, 4, 18).months_since(NaiveDate::from_ymd(2022, 4, 1)),
+    ///     MonthsDelta::Forwards(Months::new(0))
+    /// );
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2022, 5, 1).months_since(NaiveDate::from_ymd(2022, 4, 30)),
+    ///     MonthsDelta::Forwards(Months::new(1))
+    /// );
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2023, 5, 1).months_since(NaiveDate::from_ymd(2022, 4, 30)),
+    ///     MonthsDelta::Forwards(Months::new(13))
+    /// );
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2022, 4, 1).months_since(NaiveDate::from_ymd(2022, 4, 18)),
+    ///     MonthsDelta::Backwards(Months::new(0))
+    /// );
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2022, 4, 30).months_since(NaiveDate::from_ymd(2022, 5, 1)),
+    ///     MonthsDelta::Backwards(Months::new(1))
+    /// );
+    /// assert_eq!(
+    ///     NaiveDate::from_ymd(2022, 4, 30).months_since(NaiveDate::from_ymd(2023, 5, 1)),
+    ///     MonthsDelta::Backwards(Months::new(13))
+    /// );
+    /// ```
+    pub fn months_since(self, rhs: NaiveDate) -> MonthsDelta {
```

Comment:
So what was the use case for adding this?

Reply:
This is mainly for API consistency, we have `Duration` and `TimeDelta`, `Days` and `DateDelta` and here `Months` and `MonthsDelta`. 

Calculating the calendar months between two dates is often useful in financial / accounting style calculations, but it is admittedly a narrow use case, and could be better served in an external library

Reply:
Unless you have a concrete use case for this in the near future, I would propose that we leave this out for now. We already have a lot to work through in this crate, so better not to add more code unless we have a somewhat strong proof that we need/want it?

Reply:
Sounds good to me